manifests/haproxy/autoload.pp

```diff
@@ -10,7 +10,11 @@
   String  $http_edge_frontend        = 'default_fe',
 
   String  $http_edge_redirect_marker = '###http-edge-redirects###',
+  String  $http_edge_canonical_marker = '###http-edge-canonicals###',
+  String  $http_edge_nofollow_marker = '###http-edge-nofollow###',
+  String  $http_edge_status410_marker = '###http-edge-status410###',
```

Comment:
Instead of 4 variables (that could grow to 5, 7 ,... n), use a hash like `$http_edge_markers = {'redirects' => ..., }`

Reply:
done